dapr/clients/grpc/_response.py

```diff
@@ -311,3 +311,93 @@ def data(self) -> bytes:
     def data(self, val: Union[bytes, str]) -> None:
         """Sets str or bytes type data to request data."""
         self._data = to_bytes(val)
+
+
+class BulkStateItem:
+    """A state item from bulk_get_state API.
+
+    Attributes:
+        key (str): state's key.
+        data (Union[bytes, str]): state's data.
+        etag (str): state's etag.
+        error (str): error when state was retrieved
+    """
+
+    def __init__(
+            self,
+            key: str,
+            data: Union[bytes, str],
+            etag: str = '',
+            error: str = ''):
+        """Initializes BulkStateItem item from :obj:`runtime_v1.BulkStateItem`.
+
+        Args:
+            key (str): state's key.
+            data (Union[bytes, str]): state's data.
+            etag (str): state's etag.
+            error (str): error when state was retrieved
+
+        Raises:
+            ValueError: if the response data is not :class:`google.protobuf.any_pb2.Any`
```

Comment:
init does not raise any error. If needed type check data and raise error.

Reply:
removed comment.